### PR TITLE
Chat QOL features

### PR DESCRIPTION
**Pull request by Nasko-5**
my pr makes it look nicer and also scrolls the page down whenever a new message is recieved
(i have not tested this because im too lazy to figure out how to, the css works though)

### CODE REVIEW

File: chat.html [hunk 1]
```diff
@@ -80,6 +93,7 @@
             messageContainer.appendChild(messageTextElement);
             messagesContainer.appendChild(messageContainer);
             messagesContainer.scrollTop = messagesContainer.scrollHeight;
```
Comment (infernostars) [inline chat.html:95]: This line shouldn't be needed anymore as it's the one which tries to scroll

**Comment by infernostars** [comment]
Other than that, looks good to me though

**Comment by Nasko-5** [comment]
oh wait i think the service behind the chat died, uhm

File: chat.html (hunk 1 above)
Comment (Nasko-5) [reply chat.html]: i removed it

**Comment by infernostars** [comment]
can you undo the last commit

**Comment by infernostars** [comment]
#77

**Comment by Nasko-5** [comment]
i think that did it?

**Comment by infernostars** [comment]
yep. lgtm

File: chat.html (hunk 1 above)
Comment (infernostars) [reply chat.html]: looks good to me

**Review by Dishpit — CHANGES_REQUESTED** [review]
pls fix merge conflicts <3

**Comment by Nasko-5** [comment]
oh nvm someone already did something better, closing this

**Pull request closed without merging**